GameDataParser/Parsers/ConstantsParser.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Xml;
+using GameDataParser.Files;
+using Maple2.File.IO.Crypto.Common;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    public class ConstantsParser : Exporter<List<ConstantsMetadata>>
+    {
+        public ConstantsParser(MetadataResources resources) : base(resources, "constants") { }
+
+        protected override List<ConstantsMetadata> Parse()
+        {
+            List<ConstantsMetadata> constants = new List<ConstantsMetadata>();
+            foreach (PackFileEntry entry in Resources.XmlReader.Files)
+            {
+                if (!entry.Name.StartsWith("table/constants"))
+                {
+                    continue;
+                }
```

Comment:
Instead of looping trough every file, you could just get the first file with the desired name